pytripgui/canvas_vc/plotter/mpl_plotter.py

```diff
@@ -0,0 +1,85 @@
+from PyQt5 import QtCore
+from PyQt5.QtWidgets import QSizePolicy
+from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.figure import Figure
+
+from pytripgui.canvas_vc.plotter.managers import PlottingManager, BlitManager
+
+
+class MplPlotter(FigureCanvas):
+    """
+    Viewer class for matplotlib 2D plotting widget
+    """
+    def __init__(self, parent=None, width=16, height=9, dpi=100):
+        """
+        Init canvas.
+        """
+        super().__init__()
+        self.figure = Figure(figsize=(width, height), dpi=dpi, tight_layout=True)
+        self.blit_manager = BlitManager(self)
```

Comment:
Add a comment mentioning that the moment at which `BlitManager` is created is important.
If I remember correctly it has to be created once `self.figure` is defined

Reply:
Done